exercises/beer-song/example.tt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'beer_song'
+
+# Common test data version: <%= abbreviated_commit_hash %>
+class BeerSongTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.test_name %>
+    <%= test_case.skipped %>
+    expected = <<-TEXT
+<%= test_case.expected %>
+TEXT
```

Comment:
Oh, right. Yepp. I don't have strong feelings either way, but I think I'll indent the ending delimiter.